dashboard/config/linux/bits/debug.yml

```diff
@@ -3,6 +3,13 @@
 
 # Auxiliary debug configs that detect non-critical bugs.
 
+# CONFIG_RCU_EXP_CPU_STALL_TIMEOUT is not only in the linux-next branch.
+# By default it is set to a too small value, which causes tons of `rcu_preempt detected expedited stalls
+# on CPUs/tasks`.
+# TODO: turn this into a normal config once it reaches mainline.
+verbatim: |
+  CONFIG_RCU_EXP_CPU_STALL_TIMEOUT=21000
```

Comment:
Should this be in debug.yml or in base.yml?
Will these "rcu_preempt detected expedited stalls" happen if debug.yml is not included? Is there anything in debug.yml that enables this warning?

Reply:
This is an interesting question. We set configs like `RCU_CPU_STALL_TIMEOUT: 100` in x86_64, arm, arm64, etc. bits, each with a different numeric value. But the max value for `CONFIG_RCU_EXP_CPU_STALL_TIMEOUT` is already pretty low, so it should probably be the same for all platforms.

On the other hand, though, it seems that we also need to enable this sysctl in CMDLINE:
https://elixir.bootlin.com/linux/v5.17-rc5/source/kernel/sysctl.c#L2337

Otherwise the INFO messages won't result in panics and syzkaller will treat these reports as corrupted.
kernel/rcu/Kconfig.debug

Reply:
I updated the PR. I decided that it's better to keep `CONFIG_RCU_EXP_CPU_STALL_TIMEOUT=21000` and `sysctl.max_rcu_stall_to_panic=1` in just one bit file instead of tons of others. 

@dvyukov, what do you think?

Reply:
It's definitely better to keep it in one file and not duplicate.
But should it be in debug.yml or base.yml? Is this warning enabled if debug.yml is not included in a config?

Reply:
I've just run `make defconfig` on a clean branch, the resulting config already includes `CONFIG_RCU_STALL_COMMON=y`. So base.yml is a better place indeed.

Updated the PR.

Reply:
Ah, then also `sysctl.max_rcu_stall_to_panic=1` should go to base. Updating the PR once more.

Reply:
Done